cmd/documentation/root_templates/root_readme.tmpl

```diff
@@ -78,6 +78,7 @@ However, we welcome pull requests for any exchange which does not match this cri
 + WebGUI (discontinued).
 + Exchange HTTP mock testing. See [mock](/exchanges/mock/README.md).
 + Exchange multichain deposits and withdrawals for specific exchanges. See [multichain transfer support](/docs/MULTICHAIN_TRANSFER_SUPPORT.md).
++ Sonic JSON Integration. Revert back to the default Go 'encoding/json' package from the sonic library using go build tags `-tags=sonic_off`, by using the make file `make no_sonic`, or when compiling for a 32-bit architecture (GOARCH=386).
```

Comment:
* Still don't think `Current Features` is the right place for this.. Sonic isn't a feature; it's an implementation choice.
* Even if this does stay here, "Revert back..." instructions is definitely not right here.

I suggest this gets moved to part of `Compiling instructions`

Reply:
[cya mate](https://github.com/thrasher-corp/gocryptotrader/pull/1623/commits/d49cf8e43b9e167cce14b4b6cacb7dbf5637fdf5)